src/detect-engine-build.c

```diff
@@ -1079,8 +1064,7 @@ static int RulesGroupByProto(DetectEngineCtx *de_ctx)
     return 0;
 }
 
-static int PortIsWhitelisted(const DetectEngineCtx *de_ctx,
-                             const DetectPort *a, int ipproto)
+static int PortIsPriority(const DetectEngineCtx *de_ctx, const DetectPort *a, int ipproto)
 {
     DetectPort *w = de_ctx->tcp_whitelist;
```

Comment:
Since we're changing the name to reflect the purpose better, I suggest we change `s/tcp_whitelist/tcp_priorityport` and also for UDP.

Reply:
yeah I think we should

Reply:
Thanks, both! 🙇🏽‍♀️ Coming up in the next PR